### ISSUE TITLE
Formatter option to align text in multi-line list items

### ISSUE THREAD

**Issue by Vampire** [asciidoctor/asciidoctor-intellij-plugin#1564]
Currently the formatter always puts line continuation text in a list item on the very start of the line like
```adoc
* foo
** bar
baz
```

It would be nice if there were a formatter option to toggle this, so that they are text-aligned, so in this case
```adoc
* foo
** bar
   baz
```

**Comment by ahus1** [comment]
The current formatter tries to remove as many blanks as possible to create a common format. Similar to #1565, I'm marking this as help-wanted. If no-one picks it up, and there are no votes, I might eventually close it.

**Comment by mojavelinux** [comment]
I agree. This is a very unorthodox style for AsciiDoc documents and I wouldn't encourage it. Although I can see how it improves readability, it also wastes a lot of time/effort to ensure those extra spaces are included...exactly the kind of task AsciiDoc tries to help you avoid.
